bin/aws-kinesis-agent-babysit

```diff
@@ -8,7 +8,7 @@ PIDFILE=/var/run/$DAEMON_NAME.pid
 SERVICE="service $DAEMON_NAME"
 
 function get_agent_pid() {
-  echo $(ps --ppid $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
+  echo $(ps -p $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
```

Comment:
I think --ppid is required here because we want to list processes by parent process ID. IIRC the java process is spawned by start-aws-kinesis-agent, and $PIDFILE actually notes down the process ID of start-aws-kinesis-agent